fbs/flatbuffers/base.h

```diff
@@ -0,0 +1,227 @@
+#ifndef FLATBUFFERS_BASE_H_
```

Comment:
Upgraded FlatBuffers. The headers need to match the version of the flatc compiler; the old compiler wasn't available and the source/version wasn't recorded anywhere, so we're now on the latest release.

Future maintainers will have to repeat this process unless we include a copy of the FlatBuffers repo wholesale, but it doesn't sound like there's much appetite to do that.